oap-stdlib/src/main/java/oap/http/Client.java

```diff
@@ -311,22 +311,9 @@ public Response post( String uri, byte[] content, String contentType, Map<String
 
     @SneakyThrows
     public Response uploadFile( String uri, RequestBody body, Map<String, Object> givenHeaders ) {
-        OkHttpClient client = new OkHttpClient();
-        okhttp3.Request.Builder requestBuilder = new okhttp3.Request.Builder()
-                .url( uri )
-                .post( body );
-        givenHeaders.forEach( ( name, value ) -> requestBuilder.addHeader( name, value == null ? "" : value.toString() ) );
-
-        var response = client.newCall( requestBuilder.build() ).execute();
-
-        var headers = response.headers();
-        final Stream<String> stream = Stream.of( headers.names() );
-        final List<Pair<String, String>> headersMap = stream.map( headerName -> Pair.__( headerName, headers.get( headerName ) ) ).toList();
-        var responseBody = response.body();
-        ContentType contentType = Optional.ofNullable( responseBody.contentType() ).map( mt -> ContentType.create( mt.type() + "/" + mt.subtype(), mt.charset() ) ).orElseThrow();
-        return new Response( response.code(), response.message(), headersMap,
-                contentType.toString(),
-                responseBody.byteStream() );
+        Buffer sink = new Buffer();
+        body.writeTo( sink );
+        return post( uri, sink.readByteArray(), ContentType.MULTIPART_FORM_DATA.getMimeType(), givenHeaders );
```

Comment:
Why do we need to include extra dependencies just to call one method. Why can't we use byte array simply.

Reply:
request body has only one method to get byte[]